src/migrations/20211105105509-add-predata-column-to-events.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(`ALTER TABLE events ADD COLUMN pre_data json;`, cb);
```

Comment:
We should really default to jsonb unless you depend on whitespace and order of data being the exact same on insert and select. jsonb stores it more efficiently and supports more query operations
See https://www.citusdata.com/blog/2016/07/14/choosing-nosql-hstore-json-jsonb/

Reply:
jeah, I thought of that first, but i saw that data already was json. Not sure of the consequence of alter it?